src/tests/routes/controllers/social-auth.test.js

```diff
@@ -0,0 +1,121 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+
+import { socialController } from '../../../routes/controllers/auth.controller';
+import {
+  facebookAuth,
+  twitterAuth,
+} from '../../../routes/services/passport-strategies.services';
+
+chai.use(chaiHttp);
+
+describe('Social authentication', () => {
+  let facebookTestUser;
+  let twitterTestUser;
+  describe('Passport strategies setup', () => {
+    const accessToken = '',
+      refreshToken = '',
+      token = '',
+      tokenSecret = '',
+      done = (err, data = null) => {
+        if (data) {
+          return data;
+        }
+      };
+
+    const name = {
+      givenName: 'Uncle',
+      familyName: 'Musonant',
+    };
+    it('Should authenticate user profile from facebook', async () => {
+      const profile = {
+        id: 'demoFbSocialId',
+        name,
+        emails: [{ value: 'testMail.fb.com' }],
+        username: 'testFbUsername',
+        provider: 'facebook',
+      };
+      facebookTestUser = await facebookAuth(
+        accessToken,
+        refreshToken,
+        profile,
+        done,
+      );
+      expect(facebookTestUser.dataValues.id).to.be.a('string');
+      expect(facebookTestUser.dataValues.lastName).to.eql('Musonant');
+      expect(facebookTestUser.dataValues.email).to.eql('testMail.fb.com');
+      expect(facebookTestUser.dataValues.username).to.eql('testMail.fb.com');
+    });
+    it('Should authenticate user profile from twitter', async () => {
+      const profile = {
+        id: 'demoTwSocialId',
+        name,
+        emails: [{ value: 'testMail.tw.com' }],
+        username: 'testTwUsername',
+        provider: 'twitter',
+      };
+      twitterTestUser = await twitterAuth(token, tokenSecret, profile, done);
+      expect(twitterTestUser.dataValues.id).to.be.a('string');
+      expect(twitterTestUser.dataValues.email).to.eql('testMail.tw.com');
+      expect(twitterTestUser.dataValues.username).to.eql('testMail.tw.com');
+    });
+  });
+  describe('Social authentication controller', () => {
+    describe('should process user received from passport authentication', async () => {
```

Comment:
I think you should substitute `it` for `describe` here to make the test suite readable and understandable.